index.html

```diff
@@ -179,6 +185,39 @@ <h2>CSS</h2></li>
             </ul>
            </div>
         </div>
+        <div class="contact_section">
```

Comment:
[OPTIONAL] it is always a good practice to use section tag to wrap related content good for accessibility ,semantic reasons as well as for clean organized and easy to read code.